src/OrchardCore/OrchardCore.Liquid.Abstractions/FluidOptionsParser.cs

```diff
@@ -1,18 +1,17 @@
 using Fluid;
 using OrchardCore.Environment.Shell;
 
-namespace OrchardCore.Media.AmazonS3.Helpers;
+namespace OrchardCore.Liquid.Abstractions;
 
-// This is almost the same as in OrchardCore.Media.Azure but there isn't really a good common place for it.
-internal sealed class OptionsFluidParserHelper<TOptions> where TOptions : class
+public sealed class FluidOptionsParser<TOptions> where TOptions : class
 {
     // Local instance since it can be discarded once the startup is over.
```

Comment:
Since this is now removed from where it was originally used:

```suggestion
    // Local instance since it can be discarded once the parsing is over.
```